pkg/util/validation/limits.go

```diff
@@ -335,6 +337,8 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	}
 	f.Var(&l.NotificationRateLimitPerIntegration, "alertmanager.notification-rate-limit-per-integration", "Per-integration notification rate limits. Value is a map, where each key is integration name and value is a rate-limit (float). On command line, this map is given in JSON format. Rate limit has the same meaning as -alertmanager.notification-rate-limit, but only applies for specific integration. Allowed integration names: "+strings.Join(allowedIntegrationNames, ", ")+".")
 	f.IntVar(&l.AlertmanagerMaxConfigSizeBytes, "alertmanager.max-config-size-bytes", 0, "Maximum size of configuration file for Alertmanager that tenant can upload via Alertmanager API. 0 = no limit.")
+	f.IntVar(&l.AlertmanagerMaxSilences, "alertmanager.max-silences", 0, "Maximum number of active and pending silences that a tenant can have at once. 0 = no limit.")
+	f.IntVar(&l.AlertmanagerMaxPerSilenceSizeBytes, "alertmanager.max-per-silence-size-bytes", 0, "Maximum per silence size. 0 = no limit.")
```

Comment:
Enormous nitpick: `max-per-silence-size-bytes`->`max-silence-size-bytes`
It's shorter and consistent with `max-template-size-bytes` (applies to wherever the term appears)

Reply:
Ha! You suggested it at the same time I fixed it 🙂 I changed it to make it the same as the others!